components/script/dom/bindings/reflector.rs

```diff
@@ -43,6 +43,16 @@ where
 }
 
 pub(crate) trait DomGlobalGeneric<D: DomTypes>: DomObject {
+    /// Returns the [`GlobalScope`] of the realm that the [`DomObject`] was created in.  If this
```

Comment:
By the way I don't think this is true, because in our bindings `cx` seems to be the current realm in which the method is called, not the realm where the object was created(see https://github.com/servo/servo/issues/36081).

Reply:
The realm that's active is not the same thing as the global object that is returned from the method. The JS engine returns the global that is associated with the reflector, wrapped into the active realm.